### PR TITLE
Vinu/ fix: changed balance color from red to black

### PR DESCRIPTION
**Pull request by vinu-deriv**

### CODE REVIEW

**Review by farrah-deriv — APPROVED** [review]

File: src/styles/navbar.module.scss [hunk 1]
```diff
@@ -301,3 +301,11 @@ input:checked + .slider:before {
 .slider.round:before {
   border-radius: 50%;
 }
+
+.account_header {
+  font-weight: bold;
+  color: var(--text-main);
+  :hover {
+    cursor: pointer;
```
Comment (aswathy-deriv) [inline src/styles/navbar.module.scss]: is hover needed?we are doing for the mobile right

Comment (vinu-deriv) [reply src/styles/navbar.module.scss]: no not needed, incase sometime in future if we think of using this on a website. So added it

Comment (aswathy-deriv) [reply src/styles/navbar.module.scss]: okay

**Review by aswathy-deriv — APPROVED** [review]

File: src/styles/navbar.module.scss (hunk 1 above)
Comment (vinu-deriv) [reply src/styles/navbar.module.scss]: made change

**Pull request merged**